src/DynamoPackages/Package.cs

```diff
@@ -139,6 +140,8 @@ internal IEnumerable<Assembly> NodeLibraries
         /// </summary>
         public PackageUploadRequestBody Header { get; internal set; }
 
+        public bool RequiresSignedEntryPoints { get; internal set; }
```

Comment:
does this need to be public? Does this effect the package header? Summary if it needs to be public.

Reply:
If it effects header then we need to consider if this effects package upload/download?

Reply:
Will move this to internal as well